Dockerfile

```diff
@@ -1,16 +1,15 @@
-FROM python:3-buster
+FROM amazon/aws-lambda-python:3
 
-ARG MANAGER_VERSION="2.1.1"
+ARG MANAGER_VERSION="2.1.2"
 
 # Install build dependencies
 RUN pip3 install --no-cache-dir --disable-pip-version-check --progress-bar off 'poetry==1.1.4'
 
 # Install ecs manager
 WORKDIR /home/
-RUN curl --silent --show-error --location https://github.com/trussworks/terraform-aws-lambda-ecs-manager/archive/"$MANAGER_VERSION".zip > manager.zip \
- && unzip manager.zip
+COPY . .
 
 WORKDIR /home/terraform-aws-lambda-ecs-manager-"$MANAGER_VERSION"
 RUN poetry build && pip3 install --no-cache-dir --disable-pip-version-check dist/functions-"$MANAGER_VERSION"-py3-none-any.whl
```

Comment:
GitHub isn't letting me do this with the suggested change feature, but I think you want to replace the `WORKDIR /home/` on line 9 with the WORKDIR instruction from line 12.

```
# Install ecs manager
WORKDIR /home/terraform-aws-lambda-ecs-manager-"$MANAGER_VERSION"
COPY . .

RUN poetry build && pip3 install --no-cache-dir --disable-pip-version-check dist/functions-"$MANAGER_VERSION"-py3-none-any.whl
``